klient/k8s/res/resource.go

```diff
@@ -41,11 +41,11 @@ type Resources struct {
 	client cr.Client
 }
 
-// Res instantiates the controller runtime client
+// Resource instantiates the controller runtime client
 // object. User can get panic for belopw scenarios.
 // 1. if user does not provide k8s config
 // 2. if controller runtime client instantiation fails.
-func Res(cfg *rest.Config) *Resources {
```

Comment:
This is a constructor, so it should probably be named something like `NewResources()`.

Reply:
I missed package renaming. Thanks! 
I have renamed package to resource. Also, I have changed the constructor to `NewResource`